tests/nested/manual/remodel-simple/task.yaml

```diff
@@ -1,8 +1,8 @@
 summary: verify a simple UC20 remodel
 details: |
   Execute a simple remodel scenario under the test-snapd brand. Both models use
-  the same test-snapd-whitebox device view store. The new model requires
-  additional snaps that are present in the test-snapd-whitebox store.
+  the same test-snapd-from device view store. The new model requires
```

Comment:
This changes are unrelated.

Reply:
I see it is intentional to just check with woke the inclusive naming in the changed files.

This is the list of findings in case all the project files are analyzed
https://paste.ubuntu.com/p/9CtxVWgNgP/

Reply:
OK, but the inclusive naming check is not blocking, so we don't necessarily have to fix the things it reports.